### ISSUE TITLE
Design GUI for starting page

### ISSUE THREAD

**Issue by i315315** [i315315/amos2022ws02-automotive-test-app#17]
## User story
1. As a User
2. I want to be able to enter an area in the test app where all use cases for navigation are collected
3. So that I can easily switch between the use cases.

With this User story the graphical user interface of the starting page of the app is to be designed.

## Acceptance criteria
* The graphical user interface of the starting page of the app is designed.
* An XML for the GUI of the starting page has been created in Android studio.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
